frontend/src/modules/member/pages/member-merge-suggestions-page.vue

```diff
@@ -1,36 +1,226 @@
 <template>
   <app-page-wrapper>
-    <app-back-link
-      class="mt-1 mb-4"
-      :default-route="{
-        path: '/contributors',
-        query: { projectGroup: selectedProjectGroup?.id },
-      }"
-    >
-      <template #default>
-        Contributors
-      </template>
-    </app-back-link>
-    <h4 class="text-xl font-semibold leading-9 mb-1">
-      Merging suggestions
-    </h4>
-    <div class="text-xs text-gray-600 pb-6">
-      LFX is constantly checking your community for duplicate contributors.
-      Here you can check all the merging suggestions.
-    </div>
+    <div>
+      <app-back-link
+        :default-route="{
+          path: '/contributors',
+          query: { projectGroup: selectedProjectGroup?.id },
+        }"
+        class="font-semibold"
+      >
+        <template #default>
+          Contributors
+        </template>
+      </app-back-link>
+      <div class="flex items-center">
+        <h4 class="text-xl font-semibold leading-9">
+          Merging suggestions <span class="font-light text-gray-500">({{ total }})</span>
+        </h4>
+        <el-tooltip
+          placement="top"
+          content="LFX is constantly checking your community for duplicate contributors. Here you can check all the merging suggestions."
+        >
+          <i class="ri-question-line text-lg text-gray-500 flex items-center ml-2 h-5" />
+        </el-tooltip>
+      </div>
+
+      <cr-table class="mt-6">
+        <thead>
+          <tr>
+            <th colspan="2">
+              Contributors
+            </th>
+            <th>Confidence level</th>
+          </tr>
+        </thead>
+        <tbody>
+          <tr v-for="(suggestion, si) of mergeSuggestions" :key="si">
+            <td>
+              <div class="flex items-center gap-2">
+                <app-avatar
+                  :entity="suggestion.members[0]"
+                  size="xs"
+                />
+                <p class="text-xs leading-5 font-semibold whitespace-nowrap">
+                  {{ suggestion.members[0].displayName }}
+                </p>
+              </div>
+            </td>
+            <td>
+              <div class="flex items-center gap-2">
+                <i class="text-xl ri-subtract-line text-gray-300" />
+                <app-avatar
+                  :entity="suggestion.members[1]"
+                  size="xs"
+                />
+                <p class="text-xs leading-5 font-semibold whitespace-nowrap">
```

Comment:
We should be able to link the member to the profile page on a new tab. Such as we have in the dialog